travis/travis_install_nightly

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-pip install -q QUnitSuite flake8 coveralls pylint Click > /dev/null 2>&1
+pip install -q QUnitSuite flake8 coveralls Click oca-pylint-plugin > /dev/null 2>&1
```

Comment:
IMO the oca-pylint-plugin is a component of this PR, it's only that we agree that is best for it to be hosted in a different repo.
It should  be validated by reviewers before if can be used.